cryptofeed/exchange/bitmex.py

```diff
@@ -502,3 +508,36 @@ async def subscribe(self, websocket):
         for i in range(0, len(chans), 10):
             await websocket.send(json.dumps({"op": "subscribe",
                                              "args": chans[i:i + 10]}))
+
+    @staticmethod
+    async def _authenticate(conn):
+        """Send API Key with signed message.
+
+        https://github.com/BitMEX/sample-market-maker/blob/master/test/websocket-apikey-auth-test.py"""
+        BITMEX_API_KEY = os.environ.get("BITMEX_API_KEY")
+        BITMEX_API_SECRET = os.environ.get("BITMEX_API_SECRET")
+
+        if not BITMEX_API_KEY or not BITMEX_API_SECRET:
+            LOG.warning("BITMEX: no authentication - missing environment variables: BITMEX_API_KEY and BITMEX_API_SECRET")
```

Comment:
inside the exchange files when referencing the exchange name please use self.id. Currently they are all just uppercase strings, but should the format change i dont want to have to hunt down aberrations like this

Reply:
Done :+1: